docker/compose.yml

```diff
@@ -0,0 +1,16 @@
+services:
+  faktory:
+    image: contribsys/faktory:latest
+    command: "/faktory -b :7419 -w :7420"
+  nginx:
+    depends_on:
+      - faktory
+    image: nginx:1.25-alpine
+    ports:
+      - "127.0.0.1:7419:7419"
```

Comment:
This is for the standard integration tests (those not using `TlsStream`, i.e. the absolute majority) to run as usual.